profiles/arch/amd64/use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Yiyang Wu <xgreenlandforwyy@gmail.com> (2022-10-01)
+# ROCm works here
```

Comment:
I think you need a better comment here.
Like: "ROCm is currently only supported on the amd64 arch"
Same with the comment in the `base` file